src/App.css

```diff
@@ -1,11 +1,22 @@
-*{
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+:root {
+  --main-color: rgba(239, 155, 40, 1);
+}
+
+:root {
```

Comment:
it can be anything else as well. I prefer having two classes  `.dark` and `.light` and give one of them to a parent element and switch between them when switching between modes.